kctf-operator/pkg/apis/kctf/v1alpha1/challenge_types.go

```diff
@@ -102,27 +109,32 @@ type ChallengeSpec struct {
 	PodTemplate *corev1.PodTemplate `json:"podTemplate"`
 
 	// PersistentVolumeClaim are used to determine how much resources the author requires for its challenge
-	// Default value: size 1Gb
 	// +optional
-	PersistentVolumeClaims *corev1.PersistentVolumeClaimList `json:"persistentVolumeClaim,omitempty"`
+	Claims []string `json:"claims,omitempty"`
```

Comment:
I would prefer name "Volumes" here or "VolumeClaims", or even leave the old name. The reason is that authors probably don't know anything about k8s and the word "claims" just doesn't tell a person anything and can confuse people.